google_photos_takeout_helper/__main__.py

```diff
@@ -73,20 +70,7 @@ def main():
     )
     args = parser.parse_args()
 
-    print('DISCLAIMER!')
-    print("Before running this script, you need to cut out all folders that aren't dates")
-    print("That is, all album folders, and everything that isn't named")
-    print('2016-06-16 (or with "#", they are good)')
-    print('See README.md or --help on why')
-    print("(Note: not ALL photos found in album folders will be duplicated in date folders. You should maintain a separate backup of the original Google Takeout folder/zip to ensure you don't lose any photos. See [Issue #22](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/issues/22) for more details)")
-    print()
-    print('Type "yes i did that" to confirm:')
-    response = input()
-    if response.lower() == 'yes i did that':
-        print('Heeeere we go!')
-    else:
-        print('Ok come back when you do this')
-        exit(-2)
+    print('Heeeere we go!')
```

Comment:
# :heart_eyes: 

Feels so good to just delete it

Reply:
I'll also update the ReadMe later.